.github/workflows/go.yml

```diff
@@ -83,6 +81,53 @@ jobs:
           path: |
             k0s
           key: build-${{env.cachePrefix}}
+
+  build_windows:
+    name: Build windows
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get PR Reference and Set Cache Name
+        run: |
+          PR_NUMBER=$(echo ${GITHUB_REF} | cut -d / -f 3 )
+          echo "cachePrefix=k0s-win-${PR_NUMBER}-${{ github.sha }}" >> $GITHUB_ENV
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.13
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        name: Go modules cache
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-win-
+
+      - name: Bindata cache
+        uses: actions/cache@v2
+        id: generated-bindata
+        with:
+          path: |
+            .bins.windows.stamp
+            embedded-bins/staging/windows/bin/
+            bindata_windows
+            pkg/assets/zz_generated_offsets_windows.go
+            embedded-bins/Makefile.variables
+
+          key: ${{ runner.os }}-embedded-bins-win-${{ hashFiles('**/embedded-bins/**/*') }}
+          restore-keys: |
+            ${{ runner.os }}-embedded-bins-win-${{ hashFiles('**/embedded-bins/**/*') }}
+
+      - name: Build
+        run: make k0s.exe
+
+      - name: Run unit tests
+        run: make check-unit
```

Comment:
I don't think we need to run unit tests on win build job. those are anyway running with GOOS=linux, so there's no benefit AFAIK